ed_sensor_integration/src/laser/laser_plugin.h

```diff
@@ -38,14 +40,15 @@ class LaserPlugin : public ed::Plugin
 
     std::queue<sensor_msgs::LaserScan::ConstPtr> scan_buffer_;
 
-    tf::TransformListener* tf_listener_;
+    tf2_ros::Buffer tf_buffer_;
+    std::unique_ptr<tf2_ros::TransformListener> tf_listener_;
 
     LaserUpdater updater_;
 
     void scanCallback(const sensor_msgs::LaserScan::ConstPtr& msg);
 
     /**
-     * @brief update update the worldmodel based on a novel laserscan message.
+     * @brief Update the worldmodel based on a novel laserscan message.
```

Comment:
```suggestion
     * @brief update Update the worldmodel based on a novel laserscan message.
```

Reply:
Not needed to start the brief with the function name